crates/jwk/src/lib.rs

```diff
@@ -102,6 +102,10 @@ impl FromStr for JWK {
     }
 }
 
+pub fn from_bytes(bytes: &[u8]) -> Result<JWK, serde_json::Error> {
```

Comment:
Can you make this a constructor of `JWK`? And also add a `TryFrom<&[u8]>` implementation?

Reply:
@timothee-haudebourg - ok.  I moved it down to line 313, within the JWK implementation.  I *hope* that's what you mean by make it a constructor.  
And I added the TryFrom implementation too.